Documentation/bpf/toolchain.rst

```diff
@@ -1775,7 +1775,7 @@ of all details, but enough for getting started.
   It basically means that the ``bpf`` classifier does not need to call into external
   tc action modules, which are not necessary for ``bpf`` anyway, since all packet
   mangling, forwarding or other kind of actions can already be performed inside
-  the single BPF program which is to be attached, and is therefore significantly
+  the single BPF program, which is to be attached, and is therefore significantly
```

Comment:
No, adding the comma here reads as if “the single BPF program” is “significantly faster”, when the real subject is “all packet mangling, forwarding or other kind of actions”.

Let's remove “which is to be attached” entirely, it doesn't bring any useful info here:

```suggestion
  a single BPF program, and is therefore significantly
```